crypto/rand/rand_lcl.h

```diff
@@ -128,7 +128,12 @@ extern RAND_BYTES_BUFFER rand_bytes;
 extern RAND_DRBG rand_drbg;
 extern RAND_DRBG priv_drbg;
 
-/* How often we've forked (only incremented in child). */
+/*
+ * A "generation count" of forks.  Increment-only, and only ever written to
+ * in the child process of the fork, which is guaranteed to be single-threaded.
+ * As such, no locking is needed for normal (read) accesses, since the rest of
+ * fork processing is assumed to introduce the necessary memory barriers.
```

Comment:
My thinking was that the pthread library's fork handling is part of the broader "fork processing" umbrella, and the pthread implementation has to get pretty intertwined with libc anyway, but I can reword this if you want.